app/src/main/java/com/x8bit/bitwarden/ui/tools/feature/generator/GeneratorViewModel.kt

```diff
@@ -260,7 +268,11 @@ class GeneratorViewModel @Inject constructor(
 
     private fun loadOptions() {
         when (val selectedType = state.selectedType) {
-            is Passcode -> loadPasscodeOptions(selectedType = selectedType, usePolicyDefault = true)
+            is Passcode -> loadPasscodeOptions(
+                selectedType = selectedType,
+                usePolicyDefault = false,
```

Comment:
Hmmm, why is this `false` now?

Reply:
Otherwise it will ignore the `selectedType` and always default to password

Reply:
For added context on `usePolicyDefault`: Recently the policy got renamed and likely needs to be adapted on the mobile clients: 
Admin Console: https://github.com/bitwarden/clients/pull/10275
Consumers (web clients): https://github.com/bitwarden/clients/pull/10277

Reply:
Related Jira ticket: https://bitwarden.atlassian.net/browse/PM-10671

Reply:
Ok, just so I understand, the `usePolicyDefault` will soon be renamed and still used as an override to remembering the user's preference? So for now we should just leave it as a parameter and know that it's going to change soon?

Reply:
`usePolicyDefault` can be completely removed. The policy never defaults the password type; it overrides it completely.

The new policy flag is named `overridePasswordType`, and has 3 possible values:

* `null` means that the user's last selection should be applied.
* `"password"` means that the generated type should always use the password algorithm.
* `"passphrase"` means that the generated type should always use the passphrase algorithm.

When the override type is active, the user does not get to choose. They have to use the value set by the policy.

Reply:
Ok, got it! I can remove the `usePolicyDefault` parameter now. Adding the `overridePasswordType` parameter sounds like it should be a separate PR, probably associated with that ticket PM-10671?

Reply:
I don't have a horse in that race. It seems like it's a small change from what you have, but I'm not too familiar with this code. I'll leave how to handle the paperwork to you. 😄